Makefile

```diff
@@ -42,26 +45,21 @@ ifeq ($(INTEL_TDX), 1)
 CARGO_OSDK_ARGS += --features intel_tdx
 endif
 
-CARGO_OSDK_ARGS += --boot.loader="$(BOOT_LOADER)"
-CARGO_OSDK_ARGS += --boot.protocol="$(BOOT_PROTOCOL)"
-CARGO_OSDK_ARGS += --qemu.machine="$(QEMU_MACHINE)"
+CARGO_OSDK_ARGS += --bootloader="$(BOOT_LOADER)"
+CARGO_OSDK_ARGS += --boot_protocol="$(BOOT_PROTOCOL)"
 
-ifeq ($(QEMU_MACHINE), microvm)
-CARGO_OSDK_ARGS += --select microvm
+ifneq ($(SCHEMA), "")
+CARGO_OSDK_ARGS += --schema $(SCHEMA)
 endif
 
 # To test the linux-efi-handover64 boot protocol, we need to use Debian's
 # GRUB release, which is installed in /usr/bin in our Docker image.
 ifeq ($(BOOT_PROTOCOL), linux-efi-handover64)
-CARGO_OSDK_ARGS += --boot.grub-mkrescue=/usr/bin/grub-mkrescue
-endif
-
-ifeq ($(EMULATE_IOMMU), 1)
-CARGO_OSDK_ARGS += --select iommu
+CARGO_OSDK_ARGS += --grub-mkrescue=/usr/bin/grub-mkrescue
 endif
 
 ifeq ($(ENABLE_KVM), 1)
-CARGO_OSDK_ARGS += --qemu.args="--enable-kvm"
+CARGO_OSDK_ARGS += --qemu_args+="--enable-kvm"
```

Comment:
Why changing `=` as `+=`?

The `=` can mean if the key already exists (for some keys), we will override the value. But `+=` looks like we will only append value. Further, it may be somewhat hard to parsing the values, right?

Reply:
yep, maybe this is more awkward than the original qemu_args="..."